libsolidity/codegen/CompilerContext.h

```diff
@@ -219,6 +227,11 @@ class CompilerContext
 		OptimiserSettings const& _optimiserSettings = OptimiserSettings::none()
 	);
 
+	/// If m_revertStrings is Debug, @returns inline assembly code that
+	/// stores @param _message in memory position 0 and reverts.
+	/// Otherwise returns "revert(0, 0)".
+	std::string revertString(std::string const& _message = "") const;
```

Comment:
`revertWithReasonIfDebug`? `revertWithReason`?

Reply:
I'd call it `revertReasonIfDebug` since this doesn't append to the context, only returns the code.